examples/02-registered-shopper-auth-token.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * Get authorization token for a registered customer
+ * Usage: ts-node examples/02-registered-shopper-auth-token.ts
+ * Note: Replace configuration parameters and user credentials before running
+ */
+import { ClientConfig, Customer } from "../dist";
+import {
+  getObjectFromResponse,
+  ResponseError,
+  ShopperToken,
+  stripBearer,
+} from "@commerce-apps/core";
+import ShopperCustomers = Customer.ShopperCustomers;
+
+//client configuration parameters
+const clientConfig: ClientConfig = {
+  parameters: {
+    clientId: "your-client-id",
+    organizationId: "your-org-id",
+    shortCode: "your-short-code",
+    siteId: "your-site-id",
+  },
+};
+
+//customer credentials
+const username = "username";
+const password = "password";
+
+/**
+ * Get token for the registered customer
+ *
+ * @returns authorization token
+ */
+async function getRegisteredShopperToken(): Promise<
+  ShopperToken<ShopperCustomers.Customer>
+> {
+  const credentials = `${username}:${password}`;
```

Comment:
I believe username and password need to be treated with `encodeURIComponent` before being encoded with base64 again. It sounds redundant, but that's what it says here: https://tools.ietf.org/html/rfc6749#page-16

Reply:
I have checked with our API team and confirmed that this API is not designed per OAuth2.0 standards. Auth header should be ‘Basic ’ + base64 (username + ‘:’ + password).

Reply:
Interesting. During the online workshop we discovered that passwords containing % don't work. Maybe that's been changed. Or I am confusing the endpoint with a different full Oauth2 thing in the SFCC arena.